waltid-services/waltid-verifier-api/src/main/kotlin/id/walt/verifier/Main.kt

```diff
@@ -22,6 +23,7 @@ suspend fun main(args: Array<String>) {
                     registerResolver(LocalResolver())
                     updateResolversForMethods()
                 }
+                ServiceCommons.initDevMode()
```

Comment:
same concern as with issuer-api

Reply:
init-dev-mode is executed now at the service-commons level, on `initFeatures`, being called right after `ServiceInitialization` is invoked, only when the feature is enabled, failing silently with a waning in cases of I/O issues with the config file